spec/grid_spec.rb

```diff
@@ -1,6 +1,30 @@
 require_relative "../lib/grid"
 
 RSpec.describe Grid do
+  describe "#available_coordinates" do
+    it "returns coordinates that have not been used" do
+      grid = Grid.new("A1" => "X", "B2" => "O", "C3" => "X")
+
+      expect(grid.available_coordinates).to match_array %w[A2 A3 B1 B3 C1 C2]
```

Comment:
This test doesn't feel good either. The data escaping this way feels like it could lead to too much intimate knowledge/coupling.